requirements/base.in

```diff
@@ -18,3 +18,7 @@
 -e file:.
 pyrsistent>=0.16.1,<0.17
 zipp==3.4.1
+click
+py>=1.10.0
+urllib3>=1.26.5
+aiohttp>=3.7.4
```

Comment:
Is this used anywhere? It's not evident per [this](https://github.com/apache/superset/search?q=aiohttp) search.